Content.Shared/Random/Systems/EmoteRandomSystem.cs

```diff
@@ -0,0 +1,44 @@
+using Robust.Shared.Random;
+using Robust.Shared.Timing;
+
+namespace Content.Shared.Random;
+
+public sealed class EmoteRandomSystem : EntitySystem
+{
+    [Dependency] private readonly IRobustRandom _robustRandom = default!;
+    [Dependency] private readonly IGameTiming _timing = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+    }
```

Comment:
wut

Reply:
this was removed, since there is a pr working on a separate system.